test/parser.js

```diff
@@ -113,7 +113,10 @@ test('Parse Trans component', (t) => {
                 "key3": "This is a <1>test</1>",
                 "key4": "You have <1>{{count}}</1> apples",
                 "key5": "You have <1>one <1>very</1> bad</1> apple",
-                "key6": "This is a <1><0>{{test}}</0></1>"
+                "key6": "This is a <1><0>{{test}}</0></1>",
+                "key7 default": "key7 default",
+                "key8 default <1>{{count}}</1>": "key8 default <1>{{count}}</1>",
+                "We can use Trans without i18nKey=\"\" as well!": "We can use Trans without i18nKey=\"...\" as well!"
```

Comment:
For the moment, I just deleted the dots, but maybe in the future, it's worth adding an escape character to the parser(\.)

Reply:
For this case, I would suggest keeping the original string without removing dots, and i18next should be initialized with `{ nsSeparator: false, keySeparator: false }` to disable namespace separator and key separator when using content as keys.